unblob/cli.py

```diff
@@ -23,16 +25,46 @@ def show_external_dependencies(
     if not value or ctx.resilient_parsing:
         return
 
-    dependencies = get_dependencies(ALL_HANDLERS)
+    plugin_manager = ctx.params["plugin_manager"]
+    handlers = ctx.params["handlers"]
+    plugins = ctx.params.get(
+        "plugins_path"
+    )  # may not exist, depends on parameter order...
+    handlers = add_handlers_from_plugins(plugin_manager, handlers, plugins)
+
+    dependencies = get_dependencies(handlers)
     text = pretty_format_dependencies(dependencies)
     exit_code = 0 if all(dep.is_installed for dep in dependencies) else 1
 
     click.echo(text)
     ctx.exit(code=exit_code)
 
 
+def add_handlers_from_plugins(
+    plugin_manager: UnblobPluginManager,
+    handlers: Handlers,
+    plugin_path: Optional[Path],
+):
+    if plugin_path:
+        plugin_manager.import_path(plugin_path)
+
+    plugin_manager.load_setuptools_entrypoints("unblob_v1")
```

Comment:
This is the version of the plugin api. If we were to introduce a breaking change, we'd need to change this as well. It may make sense to move this to a const somewhere.